backends/tfhe-cuda-backend/cuda/src/integer/radix_ciphertext.cuh

```diff
@@ -40,10 +44,9 @@ void as_radix_ciphertext_slice(CudaRadixCiphertextFFI *output_radix,
   output_radix->num_radix_blocks = end_lwe_index - start_lwe_index + 1;
   output_radix->lwe_dimension = input_radix->lwe_dimension;
   Torus *in_ptr = (Torus *)input_radix->ptr;
-  output_radix->ptr = (void *)(&in_ptr[start_lwe_index * lwe_size]);
-  output_radix->degrees = &input_radix->degrees[start_lwe_index * lwe_size];
-  output_radix->noise_levels =
-      &input_radix->noise_levels[start_lwe_index * lwe_size];
```

Comment:
Here the pointer indexing was wrong, as noise levels and degrees are u64 arrays with num_blocks values (not num_blocks * lwe_size).